sql/step_a111_settings.sql

```diff
@@ -0,0 +1,95 @@
+(NULL, 'DOC_DIR', 'text', NULL, '../doc', '../doc', 'Directory path in which user guides are stored', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'PATIENTEXTENDED', 'bool', NULL, 'FALSE', 'FALSE', 'Enable extended patient form', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'OPDEXTENDED', 'bool', NULL, 'FALSE', 'FALSE', 'Enable extended OPD form', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'LABEXTENDED', 'bool', NULL, 'FALSE', 'FALSE', 'Enable extended laboratory', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'PATIENTVACCINEEXTENDED', 'bool', NULL, 'FALSE', 'FALSE', 'Use extended patient vaccine', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'INTERNALVIEWER', 'bool', NULL, 'TRUE', 'TRUE', 'Use the internal viewer', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'INTERNALPHARMACIES', 'bool', NULL, 'FALSE', 'FALSE', 'Allow internal pharmacies', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'MERGEFUNCTION', 'bool', NULL, 'TRUE', 'TRUE', 'Enable patient merge feature', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'SMSENABLED', 'bool', NULL, 'FALSE', 'FALSE', 'Enable SMS sending', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'TELEMETRYENABLED', 'bool', NULL, 'FALSE', 'FALSE', 'Enable telemetry', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'VIEWER', 'text', NULL, '', '', 'Path to custom PDF viewer', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'RECEIPTPRINTER', 'bool', NULL, 'FALSE', 'FALSE', 'Enable bill receipt printing', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'VIDEOMODULEENABLED', 'bool', NULL, 'FALSE', 'FALSE', 'Whether to enable Video module or not', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'ENHANCEDSEARCH', 'bool', NULL, 'FALSE', 'FALSE', 'Enable enhanced search', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'XMPPMODULEENABLED', 'bool', NULL, 'FALSE', 'FALSE', 'Whether to enable XMPP module or not', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'DICOMMODULEENABLED', 'bool', NULL, 'FALSE', 'FALSE', 'Whether to enable DICOM module or not', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'DICOMTHUMBNAILS', 'bool', NULL, 'TRUE', 'TRUE', 'Enable DICOM thumbnails (previews)', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'ALLOWPRINTOPENEDBILL', 'bool', NULL, 'FALSE', 'FALSE', 'Allow open bills printing', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'ALLOWMULTIPLEOPENEDBILL', 'bool', NULL, 'FALSE', 'TRUE', 'Allow multiple open bills for a patient', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'DEBUG', 'bool', NULL, 'FALSE', 'FALSE', 'Use debug mode', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+(NULL, 'SESSIONTIMEOUT', 'number', NULL, '5', '5', 'Number in minute for session timeout', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+
+-- Not sure if we need to store MODE in the database
+-- (NULL, 'MODE', 'text', NULL, ', ', ', 'admin', NULL, NOW(), NULL, 0, 1, 1),
+
+-- Not sure if we need to store DEMODATA in the database
+-- (NULL, 'DEMODATA', 'bool', NULL, 'FALSE', 'FALSE', NULL, 'admin', NULL, NOW(), NULL, 0, 1, 1),
```

Comment:
@mwithi @dbmalkovsky please could you explain the use of these settings and tell me if we should store them in the database

Reply:
they are used in the scripts and also `APISERVER` is still experimental, you can avoid them